clients/mist_client.go

```diff
@@ -48,6 +48,10 @@ type MistClient struct {
 	TriggerCallback string
 	configMu        sync.Mutex
 	cache           *cache.Cache
+	// The cache uses a RWMutex which means multiple routines can end up with a cache miss at the same time and end up
```

Comment:
You mention `RWMutex`, but you actually use `sync.Mutex` 🙃

Reply:
I meant the cache library https://github.com/patrickmn/go-cache/blob/master/cache.go#L43

Reply:
I see. Thanks for upating the comment!